src/main/java/org/entando/kubernetes/model/bundle/descriptor/plugin/PluginDescriptor.java

```diff
@@ -84,6 +80,41 @@
      */
     private List<EnvironmentVariable> environmentVariables;
 
+    /******************************************************************
+     * private variables that can't be set from the yaml descriptor.
+     *****************************************************************/
+    private String bundleId;
+    private String fullDeploymentName;
+
+
+    public PluginDescriptor() {
+    }
+
+    // About the NOSONAR:
+    // a constructor with more than 7 parameters is discouraged, but we require it to fulfill lombok builder requests
+    // and for the same reason we can't remove the 2 unused parameters fullDeploymentName and bundleId
+    public PluginDescriptor(String descriptorVersion,
```

Comment:
Isn't enough to have 
```
@NoArgsConstructor
@AllArgsConstructor(access = AccessLevel.PRIVATE)
```
if it's not used?

Reply:
no, but I added the wrong message, the right one is about the all args constructor used to deserialize a yml resource into the plugin descriptor. I'm going to update the message, thx